src/optics/focusAtom.ts

```diff
@@ -0,0 +1,74 @@
+import { atom } from 'jotai'
+import * as O from 'optics-ts'
+import { PrimitiveAtom, SetStateAction, WritableAtom } from '../core/types'
+
+export type FocusAtom = {
+  <S, A>(
+    atom: PrimitiveAtom<S>,
+    callback: (optic: O.OpticFor<S>) => O.Prism<S, any, A>
```

Comment:
```suggestion
    callback: (optic: O.OpticFor<S>) => O.Prism<S, unknown, A>
```
Can all `any`s be `unknown`s?

Reply:
nope, unfortunately not :(

Reply:
How about changing as far as we can? Let's see.